docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1691,6 +1696,15 @@ function toggleSendVideo() {
     localMedia.cameraStream.setMuted(newMuteValue);
 
     console.log(`MeetingControls#toggleSendVideo() :: Successfully ${newMuteValue ? 'muted': 'unmuted'} video!`);
+
+    if (newMuteValue && vbgEffect?.isEnabled) {
+      handleVbg();
+      wasVBGEnabledBeforeToggle = true;
+    } else if(wasVBGEnabledBeforeToggle) {
+      handleVbg();
```

Comment:
handleVbg also handles the VBG button text. So, when the user has enabled VBG, then they mute and if the button says "Enable VBG" again, won't it be misleading?

Reply:
That's why introduced an additional check `wasVBGEnabledBeforeToggle`. It keeps the flow intact.

Reply:
I don't understand the point of this fix. What are we trying to fix here? When I enable an effect and then mute the video and unmute it I expect the effect to be applied and that is happening currently without this PR and also with this PR it still works that way.
When user mutes the video, we call setMuted(true) and that's all we're supposed to be doing. It's down to the webrtc-core lib and the effects lib to make sure that resources are not wasted when a track with an effect is muted. There should be no need for any fix at the app level. If there is any real problem with the fact that the effect is still there, then this should be fixed in webrtc-core or the effects lib.

Also, with this new code we get very weird behavior with these steps:
1. enable the effect
2. mute the video
I see now the effect button saying "Enable VBG", which is very misleading to the user, because if they don't press it and just unmute they see the effect is already enabled anyway. 

Also another scenario is broken:
1. enable effect
4. mute video
5. press "enable vbg"
6. unmute video
now you can see the video without the effect despite the fact that you've pressed "enable vbg" in step 3.

Reply:
I have removed this change as the issue needs to be fixed in either the effect package or webrtc-core. Have created a separate JIRA - https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-502882